appyx-interactions/common/src/commonMain/kotlin/com/bumble/appyx/interactions/sample/Children.kt

```diff
@@ -141,5 +151,16 @@ fun SampleElement(
             color = Color.Black,
             fontWeight = FontWeight.Bold
         )
+
+        val value = getMotionPropertyRenderValue<DpOffset, Position>()
+        if (value != null) {
+            Text(
+                modifier = Modifier.align(Alignment.Center),
+                text = "${value.x}, ${value.y}",
+                fontSize = 21.sp,
+                color = Color.Black,
+                fontWeight = FontWeight.Bold
+            )
+        }
```

Comment:
I'd rather not merge this as it will affect a lot of demos. Could you maybe create a standalone demo where this (and any other property) are displayed as text?

Reply:
done